src/antsibull_core/vendored/collections.py

```diff
@@ -103,6 +103,7 @@ def count(seq):
     deprecated and replaced when support for Python < 2.7 is dropped.
     """
     if not is_iterable(seq):
+        # pylint:disable-next=broad-exception-raised
         raise Exception('Argument provided  is not an iterable')
```

Comment:
This should raise a TypeError, but I guess we don't want to modify vendored code